code/modules/clothing/clothing.dm

```diff
@@ -438,7 +438,7 @@ SEE_PIXELS// if an object is located on an unlit area, but some of its pixels ar
 BLIND     // can't see anything
 */
 
-/proc/generate_female_clothing(index,t_color,icon,type)
+/proc/generate_female_clothing(index, t_color, icon, type, greyscale_colors)
```

Comment:
The greyscale_colors arg isn't needed right?

Reply:
Ah right, thanks